### PR TITLE
Extend the font metadata in accordance with the 2024 targets

### PR DESCRIPTION
**Pull request by IvanUkhov**
See https://docs.google.com/document/d/1ljEHbDvXComXnW5s_EXZ0nM3_JCLnYr28Xrcf0YYtP8.

### CODE REVIEW

File: .github/workflows/requirements.txt [hunk 1]
```diff
@@ -1,21 +1,22 @@
-pylint
```
Comment (IvanUkhov) [inline .github/workflows/requirements.txt:1]: Just sorting.

File: .github/workflows/wptagent_test.yml [hunk 2]
```diff
@@ -36,6 +36,9 @@ jobs:
         python3 -m pip install --upgrade pip
         pip3 install -r .github/workflows/requirements.txt
 
+    - name: Analysing the code with Black
+      run: black --include font --check .
```
Comment (IvanUkhov) [inline .github/workflows/wptagent_test.yml:40]: Apply only to font-related files to begin with. This is what I intend to touch in this PR.

**Comment by pmeenan** [comment]
Other than formatting and tests, is the only functional change that "head" was added to the output?

**Comment by IvanUkhov** [comment]
Yes, the head is the only change from that perspective.

**Pull request merged**